addons/account_edi_ubl_cii/models/account_edi_xml_cii_facturx.py

```diff
@@ -308,15 +310,15 @@ def _find_value(xpath, element=tree):
 
         invoice_date_node = tree.find('./{*}ExchangedDocument/{*}IssueDateTime/{*}DateTimeString')
         if invoice_date_node is not None and invoice_date_node.text:
-            date_str = invoice_date_node.text
+            date_str = re.sub(CLEAR_DATE_REGEXP, "", invoice_date_node.text)
```

Comment:
```suggestion
            date_str = invoice_date_node.text.strip()
```
Just `strip()` :slightly_smiling_face: We don't wan't to remove everything but numbers here, if character or whatever but indentation ends up here we need to know something is wrong to fix it.